tensorflow/c/eager/c_api_unified_experimental_test.cc

```diff
@@ -92,9 +92,254 @@ TEST_P(UnifiedCAPI, TestBasicEager) {
   TF_DeleteExecutionContext(ctx);
 }
 
+
+// MatMul Test
+TEST_P(UnifiedCAPI, TestBasicEagerMatMul) {
+  std::unique_ptr<TF_Status, decltype(&TF_DeleteStatus)> status(
+      TF_NewStatus(), TF_DeleteStatus);
+  TFE_ContextOptions* opts = TFE_NewContextOptions();
+  TF_ExecutionContext* ctx = TF_NewEagerExecutionContext(opts, status.get());
+  ASSERT_EQ(TF_OK, TF_GetCode(status.get())) << TF_Message(status.get());
+  TFE_DeleteContextOptions(opts);
+
+  ASSERT_EQ(TF_OK, TF_GetCode(status.get())) << TF_Message(status.get());
+
+  /* Want to test simple MatMul example: 
+
+    [ [0,0] ,   *   [ [0,0] ,   =   [ [0,0],
+      [0,0] ]         [0,0] ]         [0,0] ]
+
+  */
+
+  // Build an abstract input tensor.
+  int64_t dims [] = {2,2}; // Matrices will be 2 x 2
+  int num_dims = sizeof(dims)/sizeof(dims[0]); 
+
+  float vals [] = {0.0f,0.0f,0.0f,0.0f};
+  TFE_Context* eager_ctx = TF_ExecutionContextGetTFEContext(ctx,status.get());
+  TFE_TensorHandle* t = TestMatrixTensorHandleWithInput(eager_ctx, vals, dims,num_dims); //, dims[0],dims[1]);
```

Comment:
I believe the comment at the end of the line is not intentional?

Reply:
Yes, I'll delete that right now